### PR TITLE
Port of  admin freeze verb from NSS Pheonix [FEATURE] [Do Not Merge]

### PR DESCRIPTION
**Pull request by tigercat2000**
This ports the admin tool for freezing mobs and mechs from NSS Pheonix (Unbound Travels).

### CODE REVIEW

File: code/modules/admin/verbs/freeze.dm [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+						M.blinded = 0
+						M.lying = 0
+						M.stat = 0
+
+						M << "<b> <font color= red>You have been unfrozen by <a href='?priv_msg=\ref[usr.client]'>[key]</a></b></font>"
+						message_admins("\blue [key_name_admin(usr)] unfroze [key_name(M)]")
+						log_admin("[key_name(usr)] unfroze [key_name(M)]")
+
+
+/client/proc/freezemecha(obj/mecha/O as obj in world)
```
Comment (Markolie) [inline code/modules/admin/verbs/freeze.dm]: I'd recommend replacing world with mecha_list. It should be much more efficient.

**Comment by Spacemanspark** [comment]
Great, but we should make it a rule that people can't fuck with frozen people. At all.
Great idea, otherwise.

**Comment by Markolie** [comment]
:100:

**Comment by tigercat2000** [comment]
I will point out that, when freezing people, it clearly displays a large flashing electric overlay with 'ADMIN' on it; Probably fairly self explanatory to not touch them

**Comment by Spacemanspark** [comment]
I know, but that doesn't stop people from doing so on UT.

**Comment by ZomgPonies** [comment]
Maybe a solution would be to make frozen people unmoveable and invulnerable for the duration they are frozen.

**Comment by Spacemanspark** [comment]
That would work.

**Comment by FalseIncarnate** [comment]
Is it possible to make them unable to be stripped while frozen, or is that covered under invulnerable?

**Comment by tigercat2000** [comment]
I would think you would just freeze anyone that is stripping/attacking/dragging the person you froze.

**Comment by Spacemanspark** [comment]
Lot easier to just make then untouchable.

**Comment by tigercat2000** [comment]
Not necessarily, but I will see what I can do.

**Comment by tigercat2000** [comment]
Going to have to avoid merging until #291 is complete- Cannot modify inventory.dm until done.

**Comment by tigercat2000** [comment]
Alright, this commit should be fine to merge- #291 was merged, therefore, can safely modify inventory.dm with minimal conflict causes. Unfortunately, due to there being no feasible way to prevent attacks via weapons, frozen people can still be injured by a greysuit with a fire extinguisher, but they can no longer be pulled, punched, grabbed, or stripped.

**Pull request merged**